src/usr/local/www/system.php

```diff
@@ -122,10 +122,10 @@ function is_timezone($elt) {
 	}
 
 	$hr_offset = substr($desc, 8);
-	$plural = $hr_offset == "1" ? "" : "s";
+	$hour_word = $hr_offset == "1" ? gettext("hour") : gettext("hours");
 
 	$timezonedesc[$idx] = $desc . " " . sprintf(gettext(
-	    "(%s hour%s %s GMT)"), $hr_offset, $plural, $direction_str);
+	    '(%1$s %2$s %3$s GMT)'), $hr_offset, $hour_word, $direction_str);
```

Comment:
Good catch, I believe we could be better using ngettext(), that is a function designed to deal with plural cases. What do you think?
http://php.net/manual/en/function.ngettext.php